### PR TITLE
[5.0] remove deprecated strings

### PR DESCRIPTION
**Pull request by brianteeman**
This PR removes language strings marked for deprecation in 5.0

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
@HLeithner there are a few more strings marked for removal in #39374 which related to `build\media_source\com_users\js\admin-users-mail.es6.js` but this file was put back in #39431 and marked to be removed in 5.0. Is that js still being removed in 5.0? If not then I should remove/change the language string comment

**Comment by HLeithner** [comment]
did @richard67 removed the files from #39431 already in a 5.0 PR?

**Comment by richard67** [comment]
> did @richard67 removed the files from #39431 already in a 5.0 PR?

@HLeithner No, I haven't done that yet. Shall I do that?

**Comment by HLeithner** [comment]
yes please

**Comment by richard67** [comment]
@HLeithner See PR #40302 .

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 996c30677ebedc73aca4450a2b2bcf9078e03591<br /><br />Code review: I can confirm that this PR covers all deprecations for 5.0 in language files which have a corresponding deprecation comment.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40296">issues.joomla.org/tracker/joomla-cms/40296</a>.</sub>

**Comment by HLeithner** [comment]
@brianteeman since @richard67 brings this up in the other PR could you create a PR against the 5.0 branch in the manual please?

https://github.com/joomla/Manual/pull/28 Add a section to the b/c incompatible file

**Comment by brianteeman** [comment]
with a pr against a pr?

**Comment by HLeithner** [comment]
> with a pr against a pr?

it's the 5.0 branch in the manual repo

**Comment by richard67** [comment]
> @brianteeman since @richard67 brings this up in the other PR could you create a PR against the 5.0 branch in the manual please?
> 
> [joomla/Manual#28](https://github.com/joomla/Manual/pull/28) Add a section to the b/c incompatible file

@brianteeman Harald means the https://github.com/joomla/Manual/blob/5.0/migrations/44-50/removed-backward-incompatibility.md file.

**Comment by brianteeman** [comment]
Thanks @richard67 

https://github.com/joomla/Manual/pull/101

**Comment by richard67** [comment]
@brianteeman Now you only need to adapt the last 2 check boxes in the "Link to documentations" section of the description. The "Pull Request link for manual.joomla.org: " should be checked, and the link to https://github.com/joomla/Manual/pull/101 should be added at the end, and the other checkbox below that should be unchecked.

File: administrator/language/en-GB/com_users.ini [hunk 1]
```diff
@@ -219,15 +219,7 @@ COM_USERS_MAIL_MASSMAIL_MAIL_TITLE="Users: Mass Mail Users"
 COM_USERS_MAIL_NO_USERS_COULD_BE_FOUND_IN_THIS_GROUP="No users could be found in this group."
 COM_USERS_MAIL_ONLY_YOU_COULD_BE_FOUND_IN_THIS_GROUP="You are the only user in this group."
 COM_USERS_MAIL_PASSWORD_RESET_DESC="Sent to a user by the &quot;Forgot your password?&quot; link eg in a login form."
-COM_USERS_MAIL_PASSWORD_RESET_TITLE="Users: Password Reset"
```
Comment (wilsonge) [inline administrator/language/en-GB/com_users.ini:222]: Think the string below is the one deprecated not the one above this line :)

**Pull request closed without merging**